include/picongpu/plugins/radiation/check_consistency.hpp

```diff
@@ -30,15 +30,15 @@ namespace plugins
 namespace radiation
 {
 
-void check_consistency(void)
+inline void check_consistency(void)
```

Comment:
Why did you not use `HINLINE` here?

Reply:
Ah, yes, I was again confused of your rules on that. Will change to `HINLINE`.